.github/workflows/ci.yml

```
@@ -40,3 +50,4 @@ jobs:
 
       - name: Build project
         run: npm run build
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)